src/VirtualMonoRepo/source-mappings.json

```diff
@@ -173,16 +173,6 @@
             "name": "xdt",
             "defaultRemote": "https://github.com/dotnet/xdt"
         },
-        {
-            // TODO: Remove the xliff-tasks mapping once the synchronization flags it as unused
-            // We no longer synchronize it but we can't remove it yet until
-            // it disappears from all of the Version.Details.xml files.
```

Comment:
I still see a couple matches: https://github.com/search?q=repo%3Adotnet%2Fdotnet+xliff-tasks+path%3AVersion.Details.xml&type=code

Reply:
Ah, yeah, also just noticed. I thought the references were there but not with the SB tag. I will open PRs in those repos

Reply:
note that these repos use arcade 8.0 which doesn't have the xliff tasks

Reply:
`aspire` was to be removed from the VMR?

Reply:
Yes: https://github.com/dotnet/source-build/issues/4735